src/ipvs/ip_vs_synproxy.c

```diff
@@ -447,24 +581,31 @@ static void syn_proxy_reuse_mbuf(int af, struct rte_mbuf *mbuf,
         struct dp_vs_synproxy_opt *opt)
 {
     uint32_t isn;
-    uint32_t tmpaddr;
     uint16_t tmpport;
-    struct iphdr *iph;
     struct tcphdr *th;
-    int ip4hlen;
+    int iphlen;
+
+    if (AF_INET6 == af)
+        /* FIXME: count exthdrs length? */
+        iphlen = sizeof(struct ip6_hdr);
```

Comment:
use skip header function?

Reply:
No. `iphlen` should not count exthdrs length because it's used for tcp checksum later. `mbuf_may_pull` is not a must here because `th` is got by 'mbuf_header_pointer'